src/PortingAssistant.Client.Analysis/Utils/CodeEntityModelToCodeEntities.cs

```diff
@@ -263,6 +263,19 @@ private static PackageVersionPair GetPackageVersionPair(Reference reference, Ext
                 s.AssemblyLocation?.EndsWith(reference.Assembly + ".dll") == true || s.Identity.Equals(reference.Assembly));
             PackageVersionPair sdk = ReferenceToPackageVersionPair(potentialSdk, PackageSourceType.SDK);
 
+            // if mscorlib, try to match namespace to sdk
+            if (reference.Assembly == "mscorlib")
```

Comment:
Should we use `string.Compare(...)` or `.ToLower()` to ignore case?

Reply:
+1

Reply:
updated to use string.Compare